tests/test_cli.py

```
@@ -0,0 +1,440 @@
+            result = cli_runner.invoke(cli, [
+                'serve',
+                '--host', '0.0.0.0',
+                '--port', '8080',
+                '--max-tries', '3'
+            ])
+            assert result.exit_code == 0
+
+    def test_validate_countries(self, cli_runner):
+        """Test countries validation."""
+        # Test valid countries
+        result = cli_runner.invoke(cli, [
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)